pkg/core/helper_test.go

```diff
@@ -603,14 +606,20 @@ func prepareContractMethodInvokeGeneric(chain *Blockchain, sysfee int64,
 func signTxWithAccounts(chain *Blockchain, tx *transaction.Transaction, accs ...*wallet.Account) {
 	scope := transaction.CalledByEntry
 	for _, acc := range accs {
+		accH, _ := address.StringToUint160(acc.Address)
 		tx.Signers = append(tx.Signers, transaction.Signer{
-			Account: acc.PrivateKey().GetScriptHash(),
+			Account: accH,
```

Comment:
`acc.Contract.ScriptHash()` should also work for both cases.

Reply:
Isn't `acc.Contract.ScriptHash()` a hash of contract's script? Here I need the contract address (hash) in the chain, and now it's not the same as the hash of the contract script.

Reply:
You are right.